### PR TITLE
Js inputs and strings input to uppercase

### PR DESCRIPTION
**Pull request by TS-Severin**

### CODE REVIEW

File: javascript/js-inputs-and-strings/js-inputs-and-strings_input-to-uppercase/js/index.js [hunk 1]
```diff
@@ -10,3 +10,12 @@ Convert the input value to uppercase when clicking the button
 Hint: Access the value by using `.value` on the input element
 
 */
+
+const textInput = document.querySelector('[data-js="first-input"]');
+const uppercaseButton = document.querySelector('[data-js="button-uppercase"]');
+
+uppercaseButton.addEventListener("click", () => {
+  const oldValue = textInput.value;
```
Comment (LuamB) [inline javascript/js-inputs-and-strings/js-inputs-and-strings_input-to-uppercase/js/index.js:18]: This line is redundant if you change the next line as follows:

File: javascript/js-inputs-and-strings/js-inputs-and-strings_input-to-uppercase/js/index.js [hunk 2]
```diff
@@ -10,3 +10,12 @@ Convert the input value to uppercase when clicking the button
...
+  const newValue = oldValue.toUpperCase();
```
Comment (LuamB) [inline javascript/js-inputs-and-strings/js-inputs-and-strings_input-to-uppercase/js/index.js:19]: const newValue = textInput.value.toUpperCase()

**Review by LuamB — APPROVED** [review]
Minor comment

File: javascript/js-inputs-and-strings/js-inputs-and-strings_input-to-uppercase/js/index.js (hunk 1 above)
Comment (TS-Severin) [reply javascript/js-inputs-and-strings/js-inputs-and-strings_input-to-uppercase/js/index.js:18]: Great, thank you!

**Pull request merged**